### PR TITLE
feat: wire up the add send image message logic into app [AR-1307]

### PR DESCRIPTION
**Pull request by gongracr**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1307" title="AR-1307" target="_blank">SubtaskAR-1307</a>  Implement send file/image use case
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Currently the media attachment option allows users to choose an image from a gallery, or to take it directly from the camera. However, after choosing it, it doesn't really do anything. With this PR, once the user chooses an image, this will be sent as an asset message. 
A method to extract the image width and height programmatically through the Android `ExifInterface` was also added, in order to populate the asset message metadata.

### Dependencies (Optional)

This PR needs to be merged after [its corresponding Kalium PR](https://github.com/wireapp/kalium/pull/355) has been merged to `develop` too.

Needs releases with:

- [x] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution N/A

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-475/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-475/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-475/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3018872)(3018872)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3018872)(3018872)-dev-release.apk)

**Review by mchenani — DISMISSED** [review]
🏞🚀

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt [hunk 1]
```diff
@@ -104,15 +103,17 @@ class ConversationViewModel @Inject constructor(
         viewModelScope.launch {
             // TODO what if conversationId is null???
             conversationId?.let { sendTextMessage(it, messageText) }
-
         }
     }
 
     fun sendAttachmentMessage(attachmentBundle: AttachmentBundle?) {
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt:109]: I know now we are only addressing the image attachments. But I think we can add a `TODO` to this function as a reminder that we will have to add a `type` property on  `AttachmentBundle` to identify the type of attachment, so later here we can call the correct usecase. Let me know what you think =)

**Review by yamilmedina — APPROVED** [review]
Great work!! the last piece of the puzzle 👏  🚀  Just added a small comment, but you can ignore it if you find it irrelevant

**Review by yamilmedina — APPROVED** [review]
🥇  💯

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-475/9/) **failed**.

**Comment by AndroidBob** [comment]
Build [12](https://10.10.124.134/job/wire-android-reloaded/job/PR-475/12/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3020717)(3020717)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3020717)(3020717)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [13](https://10.10.124.134/job/wire-android-reloaded/job/PR-475/13/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3021213)(3021213)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3021213)(3021213)-dev-release.apk)

**Pull request merged**